web/app/components/favicon.hbs

```diff
@@ -0,0 +1,6 @@
+<img
+  data-test-favicon
+  src="https://www.google.com/s2/favicons?domain={{@url}}&sz=32"
```

Comment:
I feel a bit uneasy about only using an unofficial third-party API for this, given this could seemingly just disappear at any point and then break all icons. What do you think about only making this a fallback if we don't have a flight icon for the URL?

Reply:
Yeah I share your concern so some extent. I'll shift this PR back to draft while I bulletproof it.